README.md

```diff
@@ -372,6 +374,119 @@ const keyMap = {
 };
 ```
 
+#### Setting dynamic hotkeys
```

Comment:
⬜️ 📚 Add a `... at runtime` to the end of this section title. Update all links to it.